cog/version_info.py

```
@@ -0,0 +1,145 @@
+
+    #     return embed
+
+    def embed_version_info_footer(self) -> discord.Embed:
+        """
+        Main footer of the version information embeds.
+        """
+
+        return discord.Embed(
+            color=0xFF24CF,
+            footer=discord.EmbedFooter(
+                text=datetime.datetime.now(tz=datetime.timezone.utc).strftime(
```

Ruff (0.14.10)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)